pkgversion/pkgversion.py

```diff
@@ -27,7 +28,10 @@ def get_git_repo_dir():
     repo_dir = repo_dir.strip()
     if not repo_dir:
         repo_dir = None
-    return repo_dir
+
+    country_code, encoding = locale.getdefaultlocale()
+
+    return repo_dir.decode(encoding) if repo_dir and encoding else repo_dir
```

Comment:
Yes, I can fix `.travis.yml` or we need different solution to get `str` from `bytestring`